acra/configuring-maintaining/general-configuration/acra-webconfig.md

```diff
@@ -0,0 +1,108 @@
+---
+title: acra-webconfig
+bookCollapseSection: true
+---
+
+# acra-webconfig
+
+`acra-webconfig` is simple web application that provide web UI for AcraServer's runtime configuration. Currently, it is
+under development and for now allows configuring 9 parameters of AcraServer:
+* `db_host`
+* `db_port`
+* `incoming_connection_api_port`
+* `d`
+* `poison_run_script_file`
+* `poison_shutdown_enable`
+* `zonemode_enable`
+
+{{< hint info >}}
+Parameter's description you can find on AcraServer's [command line flags]({{< ref "/acra/configuring-maintaining/general-configuration/acra_server.md#command-line-flags" >}}) page.
+{{< /hint >}}
+
+## Command line flags
+
+* `--destination_host`
+  IP or domain to AcraServer/AcraConnector daemon
+  Default is `localhost`
+
+* `--destination_port`
+  Port of AcraServer's HTTP API port or AcraConnector port that proxy API requests to AcraServer
+  Default is `9191`
+
+* `--incoming_connection_host`
+
+  Host for AcraWebconfig HTTP endpoint
+  Default is `127.0.0.1`
+
+* `--incoming_connection_port`
+
+  Port for AcraWebconfig HTTP endpoint
+  Default is `8000`
```

Comment:
```suggestion
* `--incoming_connection_port=<port>`

  Port for `acra-webconfig` to listen for HTTP connections on.
  Default is `8000`.
```

It's also possible to set this to `0` to let the OS choose a port for you, but this port is not logged anywhere, so you can only find it elsewhere. I don't think this feature should be documented then.